index.html

```diff
@@ -3,19 +3,20 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>ZOMATO</title>
+    <title>SOMATO</title>
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
 </head>
+
 <body>
     <main>
         <header>
             <nav class="top-heading flex">
-                <h1 class="heading">ZOMATO</h1>
+                <h1 class="heading">SOMATO</h1>
                 <ul class="un-list flex">
-                    <li> <a href="">Home</a> </li>
-                    <li> <a href="">menu</a> </li>
-                    <li> <a href="">Contact Us</a> </li>
+                    <li> <a href="#">Home</a> </li>
+                    <li> <a href="#food-sample">menu</a> </li>
```

Comment:
could you add javascript.void(0); rather than "" or #food-sample
so it doesnt redirect the user into top 
@Navaneethakrishanan16